### ISSUE TITLE
Craft queued ignore linked storage changes from users who didn't create the queue. 

### ISSUE THREAD

**Issue by extant1** [StrangeLoopGames/EcoIssues#9322]
Tested on a masonry table and kiln only in 7.7.1

I have a masonry table queued up with 9999 mortared stone queued by PlayerA who has not changed the linked inventories on the table. PlayerB decides he wants only a single stockpile linked to that table so all ingredients in that pile go to that project so he changes the linked storages. The craft job ignores these changes that PlayerB has done. PlayerA comes over and changes the linked storages to the single stockpile and the craft queue now reflects the changes.

**Comment by MarjieAVolk** [comment]
This is intended behavior.  The work orders that you start will only respect your own linked inventory settings.

This is designed to ensure that you can safely make your crafting tables public without worrying about giving the public access to all of your storage.  I know that this is annoying when playing cooperatively with a small group of trusted friends, since in that case you would like to all share the same storage settings.

One possible solution would be, if you are authorized on the property, then you use and edit the owner's link settings.

**Comment by extant1** [comment]
This is all still in effect with authorization.  In the above scenario PlayerB owns the claim and the machine and PlayerA is an authorized person on the claim.

**Comment by MarjieAVolk** [comment]
Yes I know; in my previous message I was suggesting adding different behavior when authorized, as a solution to your problem.

**Comment by extant1** [comment]
Sorry, I misunderstood the meaning.

**Comment by MarjieAVolk** [comment]
No problem :)  Do you think this suggestion would help you?

**Comment by extant1** [comment]
If we had to choose between either the owner or the authorized I would say the owner always should be chosen however I personally think that PlayerA shouldn't require PlayerB to make a change when authorized nor should PlayerB need PlayerA, the machine should only have a 'current state' for linked storage that is editable by the owner or authorized so that playing with friends or collaboration takes precedence.
